components/Containers/NavBar/index.tsx

```diff
@@ -31,21 +33,32 @@ type NavbarProps = {
   }>;
   languages: ComponentProps<typeof LanguageDropdown>;
   onThemeTogglerClick: () => void;
+  enablePrideLogo?: boolean;
 };
 
 const NavBar: FC<NavbarProps> = ({
   navItems,
   languages,
   onThemeTogglerClick,
+  enablePrideLogo = false,
 }) => {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
 
   return (
     <nav className={`${style.container}`}>
       <div className={style.nodeIconAndMobileItemsToggler}>
         <Link className={style.nodeIconWrapper} href="/" aria-label="Home">
-          <NodejsDark className={style.nodejsLogoDark} />
-          <NodejsLight className={style.nodejsLogoLight} />
+          {enablePrideLogo ? (
+            <>
+              <NodejsDarkPride className={style.nodejsLogoDark} />
+              <NodejsLightPride className={style.nodejsLogoLight} />
+            </>
+          ) : (
+            <>
+              <NodejsDark className={style.nodejsLogoDark} />
+              <NodejsLight className={style.nodejsLogoLight} />
+            </>
+          )}
```

Comment:
What if we combine the `NodejsDark`, `NodejsLight`, `NodejsDarkPride`, and `NodejsLightPride` SVG components into a single component, perhaps called `NodejsLogo`?

Currently, we have four different versions, and if we plan to add a special logo in the future, it would be more efficient to manage these variations within a single component. This way, we can handle the logo's conditions directly within its component, rather than managing them with conditions in the NavBar container. I believe this approach would be more understandable and easier to edit 🤔

Reply:
I'll try that no dum 😄

Reply:
Updated

Reply:
I'm not sure you addressed @canerakdas point? Afaik for what I understood it should just become one logo that encompasses all (and since they're a React component) you could simply have props that modify the parts that differ them.

Pretty much the main difference between dark/light are the stroke colours I'd assume

Reply:
I understood that the condition of variation had to take place somewhere other than in the navbar. Then I may have misunderstood the suggestion.

Reply:
> I'm not sure you addressed @canerakdas point? Afaik for what I understood it should just become one logo that encompasses all (and since they're a React component) you could simply have props that modify the parts that differ them.
> 
> Pretty much the main difference between dark/light are the stroke colours I'd assume

I didn't think of it this way, but I think it's a better idea 😄

Reply:
So i understand what you propose.
So we'd have a `NodejsLogo` component that would change colour depending on the theme for the text and press the config button for the hexagon? Which means we'd have two other hex components, one green and one pride.